data.sql

```diff
@@ -0,0 +1,29 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        weight_kg,
+        neutered,
+        escape_attempts
+    )
+
+    values (
+        'Agumon',
+        '2020-02-03',
+        10.23,
+        true,
+        0
+    ),
+     ('Gabumon', '2018-11-15', 8.00, TRUE, 2),
+        ('Pikachu', '2021-01-07', 6.00, FALSE, 3),
+        ('Devimon', '2017-05-12', 11.00, TRUE, 1),
+        ('Plantmon', '2019-05-05', 5.7, TRUE, 2),
+        ('Charmander', '2020-06-08', 8.10, FALSE, 3),
+        ('Angemon', '2015-02-12', 15.00, TRUE, 1),
+        ('Boarmon', '2016-06-07', 20.00, TRUE, 4),
+        ('Blossom', '2017-10-13', 17.00, FALSE, 2),
+        ('Ditto', '2018-01-21', 4.00, TRUE, 1),
```

Comment:
- [ ] You are currently adding the wrong data to this file. Kindly ensure you add the data in the requirements below:
<img width="1009" alt="Screenshot 2023-03-16 at 09 01 04" src="https://user-images.githubusercontent.com/64774936/225552352-34c45756-9c2c-4278-a231-a36fcf3cc875.png">